tools/modules/software/module_netalertx.sh

```
@@ -0,0 +1,85 @@
+	["module_netalertx,port"]="20211"
+	["module_netalertx,arch"]="x86-64 arm64 armhf"
+)
+#
+# Module netalertx
+#
+function module_netalertx () {
+	local title="netalertx"
+	local condition=$(which "$title" 2>/dev/null)
+
+	if pkg_installed docker-ce; then
+		local container=$(docker container ls -a | mawk '/netalertx?( |$)/{print $1}')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)